pytensor/tensor/basic.py

```diff
@@ -3084,87 +3084,72 @@ def flatten(x, ndim=1):
     return x_reshaped
 
 
-def tile(x, reps, ndim=None):
+def tile(
+    A: "TensorLike", reps: Union[Sequence[int, "TensorLike"], "TensorLike"]
```

Comment:
Does this two-argument `Sequence` work? Didn't you mean `Sequence[int | "TensorLike"]` or similar?

Reply:
Yes, it should have been Union inside!